README.md

```diff
@@ -1,13 +1,22 @@
-# Grizzly: A Grafana Dashboard Manager
+# Grizzly: A tool for Managing Observability Systems
 
-A utility for managing Jsonnet dashboards - uses the Grafana REST API.
+A utility for managing various observability resources with Jsonnet. Currently supported
+are:
```

Comment:
"observability resource" reads weird to me. Maybe "...various resources for observability systems with Jsonnet...".